src/App.css

```diff
@@ -35,4 +41,42 @@
   to {
     transform: rotate(360deg);
   }
+} */
+
+.Header {
+  padding: 10px;
+}
+
+.reviews-list {
+  display: flex;
```

Comment:
Really like the Flex here, doing bits of CSS as you go along makes it less of a mountain at the end :)